src/client/protocolgameparse.cpp

```diff
@@ -3227,7 +3227,7 @@ void ProtocolGame::parsePreyData(const InputMessagePtr& msg)
         {
             const uint8_t listSize = msg->getU8();
             for (uint8_t i = 0; i < listSize; i++) {
-                getPreyMonsters(msg);
+                getPreyMonster(msg);
```

Comment:
replace all of this to getPreyMonsters(msg);
same with the other one

Reply:
That's why I made this PR, because getPreyMonsters(msg) doesn't work there. All getPreyMonsters(msg) does is get the listSize and call getPreyMonster(msg) in a loop for the list size, but the listSize loop is already read and handled in the part that you referenced, making the listSize in getPreyMonsters(msg) return an error, because it's actually a string where it's trying to get the list size.

Reply:
Yes, I said replace the whole code from listSize to end of loop.

```
const uint8_t listSize = msg->getU8();
for (uint8_t i = 0; i < listSize; i++) {
    getPreyMonster(msg);
}
```
to
```
getPreyMonsters(msg);
```
because the code is equivalent.

Reply:
oh ok I misunderstood